src/screens/LoginPage/LoginPage.tsx

```diff
@@ -129,12 +189,34 @@ function loginPage(): JSX.Element {
       toast.error(t('Please_check_the_captcha'));
       return;
     }
+    const isValidatedString = (value: string): boolean =>
+      /^[a-zA-Z]+$/.test(value);
+
+    const validatePassword = (password: string): boolean => {
+      const lengthCheck = new RegExp('^.{7,}$');
```

Comment:
I can see that you are using same regex in multiple functions, create a single variable which will contain all the required regexes. Refactor this

Reply:
Working on it

Reply:
@rishav-jha-mech 
Refactored regexp and added toast message for invalid email in all languages